CreateDB

```diff
@@ -13,32 +17,33 @@ CREATE TABLE Chat_Group (
 
 CREATE TABLE Chat_Members (
     cid INTEGER REFERENCES Chat_Group(cid),
-    uid INTEGER REFERENCES Users(uid)
+    uid INTEGER REFERENCES Users(uid),
+    joined_on TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
 
-CREATE TABLE Parcels (
-    pid serial PRIMARY KEY,
+CREATE TABLE Messages (
+    mid serial PRIMARY KEY,
     cid INTEGER REFERENCES Chat_Group(cid),
     uid INTEGER REFERENCES Users(uid),
     created_on TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
 
 CREATE TABLE Photos (
-    image bytea not null,
-    caption varchar(1000) not null
-) INHERITS(Parcels);
-
-CREATE TABLE Messages (
-    content varchar(1000) not null
-) INHERITS(Parcels);
+    image BYTEA NOT NULL,
+    mid INTEGER REFERENCES Messages(mid)
+);
```

Comment:
missing primary key

Reply:
Primary key isn't needed, a photo cannot exist without a message.

Reply:
Unless ER diagram has changed or needs changing, we should stick to it. Multimedia should have its own primary key

Reply:
Then the ER needs to be updated, a photo cannot exist without a message. La alternativa es añadir entonces la metadata del post and that's redundant data.

Reply:
How would the ER look then? The ER that I have states that the relationship between Messages and Multimedia is one-one, When I think it should be one-many since in our case, a message may contain an image/video, but an image/video could be in many messages. We'd still need a primary key for Multimedia